app/src/main/java/com/imashnake/animite/features/MainActivity.kt

```diff
@@ -61,57 +57,51 @@ class MainActivity : ComponentActivity() {
                     systemUiController.setSystemBarsColor(Color.Transparent, darkIcons = false)
                 }
 
-                val navController = rememberAnimatedNavController()
-                val navHostEngine = rememberAnimatedNavHostEngine(
-                    rootDefaultAnimations = RootNavGraphDefaultAnimations(
-                        enterTransition = { fadeIn(animationSpec = tween(1000)) },
-                        exitTransition = { fadeOut(animationSpec = tween(300)) },
-                    )
-                )
-
-                Box(Modifier.fillMaxSize()) {
-                    DestinationsNavHost(
-                        navGraph = NavGraphs.root,
-                        modifier = Modifier
-                            .align(Alignment.TopCenter)
-                            .background(MaterialTheme.colorScheme.background)
-                            .fillMaxSize(),
-                        navController = navController,
-                        engine = navHostEngine
-                    )
-
-                    val searchBarBottomPadding: Dp by animateDpAsState(
-                        targetValue = dimensionResource(R.dimen.large_padding) + if (
-                            NavigationBarPaths.values().any {
-                                it.direction == navController.appCurrentDestinationAsState().value
-                            }
-                        ) dimensionResource(R.dimen.navigation_bar_height) else 0.dp
-                    )
+                MainScreen(Modifier.fillMaxSize())
+            }
+        }
+    }
+}
 
-                    SearchBar(
-                        modifier = Modifier
-                            .align(Alignment.BottomEnd)
-                            .padding(
-                                start = dimensionResource(R.dimen.large_padding),
-                                end = dimensionResource(R.dimen.large_padding),
-                                bottom = searchBarBottomPadding
-                            )
-                            .navigationBarsPadding(),
-                        navController = navController
-                    )
+@Composable
+@OptIn(ExperimentalAnimationApi::class, ExperimentalMaterialNavigationApi::class,
```

Comment:
what kind of masochist puts 2 on 1 line and then 1 on a new line rather than
```
@OptIn(
  ExperimentalAnimationApi::class,
  ExperimentalMaterialNavigationApi::class,
  ExperimentalMaterial3Api::class
)
```

Reply:
IDE autoformatter 😩 
There's only 2 now so it's not an issue anymore